### DISCUSSION TITLE
multiple images for inference with PPStructure

### DISCUSSION THREAD

**Discussion by themantalope** [PaddlePaddle/PaddleOCR#9675] [Q&A]
Hi,

This might be a dumb question but I can't find documentation or an example. With the python API is there a way to submit multiple images in inference, as a batch? I have more GPU memory that I could utilize, and only processing one image at a time seems really slow.

Thanks

**Comment by yukang123** [comment]
We encounter the same problem. Could someone provide some solutions? Thanks!
